network/network_windows.go

```diff
@@ -185,6 +190,76 @@ func (nm *networkManager) newNetworkImplHnsV1(nwInfo *NetworkInfo, extIf *extern
 	return nw, nil
 }
 
+// add ipv4 and ipv6 routes (if dual stack mode) to windows Node
+// in dualstack mode, pods are created from different subnets on different nodes, gateway has to be node ip if pods want to communicate with each other
+// add routes to make node understand pod IPs come from different subnets and VFP will take decisions based on these routes to forward traffic and avoid Natting
+func (nm *networkManager) addNewNetRules(nwInfo *NetworkInfo) error {
```

Comment:
can you add UT for this function to test idempotency if not already?

Reply:
can you please explain more about this UT?

Reply:
we can verify if delete func/add func is called with expected argument we pass to executecommand. 

also we should verify if ifname, subnet, gateway is non-nil or non-empty before executing netsh cmds?